.travis.yml

```diff
@@ -52,6 +52,31 @@ matrix:
         - make
         - make runtest
 
+    # linux build with nss
+    - os: linux
+      env:
+        - TEST="linux gcc (nss)"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-6
+      before_install:
+            - wget https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_38_RTM/src/nss-3.38-with-nspr-4.19.tar.gz
```

Comment:
This is too brute-force; the link will become stale almost immediately.

It seems like the better course would be to use the `libnss3` package in Ubuntu.  That will require some tweaks to the configure / build process to specify the include and library paths directly, rather than inferring them from a base.  Making these tweaks in the Makefile manually, this seems to work fine with the Ubuntu standard NSS installation.

Based on the [CircleCI docs](https://circleci.com/docs/1.0/ios-builds-on-os-x/), it looks like you might be able to do the same thing with Homebrew on MacOS.

Reply:
I can look at that, using packages though will mean there is less control of the version that is supported.
Ideally when building against a system package will mean there is no need for --with-nss-dir .